site/app/controllers/student/SubmissionController.php

```diff
@@ -157,15 +157,15 @@ public function showHomeworkPage($gradeable_id, $gradeable_version = null) {
                 // Only show hidden test cases if the display version is the graded version (and grades are released)
                 $show_hidden = false;
                 if ($graded_gradeable != NULL) {
-                  $show_hidden = $version == $graded_gradeable->getOrCreateTaGradedGradeable()->getGradedVersion(false) && $gradeable->isTaGradeReleased();
+                    $show_hidden = $version == $graded_gradeable->getOrCreateTaGradedGradeable()->getGradedVersion(false) && $gradeable->isTaGradeReleased();
+                    $can_inquiry = $this->core->getAccess()->canI("grading.electronic.grade_inquiry", ['graded_gradeable' => $graded_gradeable]);
```

Comment:
https://github.com/Submitty/Submitty/commit/477eb4aafc742c9452c1941f5c744ab804b554a4

Reply:
what changed on this line?

Reply:
` $can_inquiry = $this->core->getAccess()->canI("grading.electronic.grade_inquiry", ['graded_gradeable' => $graded_gradeable]);` moved inside the `if`.

`graded_gradeable` doesn't always exist.

Reply:
Sorry, nevermind, I was just looking at the indentation of the first line.